src/main.rs

```diff
@@ -107,7 +108,7 @@ fn main() {
             "buildbot" => {
```

Comment:
So we could definitely do a purge of all the `buildbot` related code, but I figured I'd leave that for a follow-up PR.